ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -47,7 +47,7 @@ DefineRegex {
 If.hs_init {
 	Condition {
 		Type RegexMatch
-		Regex "(cs42l43|rt5682|rt700|rt711|rt713(-sdca)?)"
+		Regex "(rt722|cs42l43|rt5682|rt700|rt711|rt713(-sdca)?)"
```

Comment:
keep all RT stuff in a logical order instead of rt722 first?

Reply:
OK, I'll modify it.